src/renderers/native/ReactNativeFiber.js

```diff
@@ -459,12 +87,31 @@ const ReactNative = {
   },
 
   unstable_batchedUpdates: ReactGenericBatching.batchedUpdates,
+
+  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
+    // Used for Flow types
+    SyntheticEvent: require('SyntheticEvent'),
+
+    // Used as a mixin in many createClass-based components
+    NativeMethodsMixin: require('NativeMethodsMixin'),
+
+    // Used by react-native-github/Libraries/ components
+    PooledClass: require('PooledClass'), // Components/Touchable
+    ReactDebugTool: require('ReactDebugTool'), // RCTRenderingPerf, Systrace
+    ReactGlobalSharedState: require('ReactGlobalSharedState'), // Systrace
+    ReactNativeComponentTree: require('ReactNativeComponentTree'), // InspectorUtils, ScrollResponder
+    ReactNativePropRegistry: require('ReactNativePropRegistry'), // flattenStyle, Stylesheet
+    ReactPerf: require('ReactPerf'), // ReactPerfStallHandler, RCTRenderingPerf
```

Comment:
Should we put `ReactPerf` and `ReactDebugTool` imports under `__DEV__`?
I think they are currently only imported from DEV-only modules in RN.